api/users/signals.py

```diff
@@ -0,0 +1,13 @@
+import warnings
+
+from django.urls import reverse
+
+
+def warn_insecure(sender, **kwargs):
```

Comment:
Can we write a test for this? I'm not sure if the `caplog` fixture will pick up the `warning` but maybe there's another way to do that if not?

Reply:
The problem is that the warning is triggered by the signal and is emitted before the test, so things like caplog, recwarn will not work with a test, unless we find a way to trigger the signal from within a test.

Reply:
Or we can just test it like a normal function

Reply:
Yeah, exactly, I just want to test the functionality in the function itself. We can defer the rest to Django :)